www_src/pages/style-guide/style-guide.jsx

```diff
@@ -34,6 +35,20 @@ var StyleGuide = React.createClass({
       this.setState({loading: false});
     }, 3000);
   },
+  showModal: function () {
+    dispatcher.fire('modal:show', {
+      config: {
+        header: 'Project Remix',
+        body: 'This is your copy of My Yam Fries Recipe. You can add or change anything. The original will stay the same. Have fun!',
+        attribution: 'fancyunicorn',
+        icon: '../../img/tinker.png',
```

Comment:
Might be a good idea to add `../../img/` in the component, since this will help reduce path errors